Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/Mappings.kt

```diff
@@ -411,9 +411,9 @@ fun mirror(declarationMapper: DeclarationMapper, type: Type): TypeMirror = when
             !type.def.isAnonymous -> {
                 val baseTypeMirror = mirror(declarationMapper, type.def.baseType)
                 TypeMirror.ByValue(
-                        Classifier.topLevel(pkg, kotlinName + "Var").type,
+                        Classifier.topLevel(pkg, kotlinName + "Var").typeAbbreviation(baseTypeMirror.pointedType),
```

Comment:
Doesn't seem to work if base type is `typedef`-based too.
Same below (3 times).

Reply:
I was a bit wrong. In Kotlin terminology, _underlyingType_ is a RHS of typealias declaration and _expandedType_ is a fully substituted type. I'll rename field accordingly.
Expanded type (which is required for metadata writing) will be possible to get by recursive expansion of `AbbreviationStubType`.